pepper-apis/scripts/dsm/tests/pecgs-phase4-participant.sh

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
+set -x
+
```

Comment:
```suggestion
# This script is a utility to perform all of the actions needed on a participant so that the "Shared Learnings" tab becomes enabled in DSM.
```

Reply:
@denniscunningham - Is this what you were looking for?